frontend/app/src/features/room/components/info/InfoBar.tsx

```diff
@@ -0,0 +1,24 @@
+import { HStack } from "@chakra-ui/react";
+import { DifficultyBadge, TopicBadge } from "@/features/room";
+import { DifficultyType } from "@/constants/question";
+import { UserInfo } from "@/components/UserInfo/UserInfo";
+
+interface InfoBarProps {
+  difficulty: DifficultyType;
+  topic: string;
+}
+
+export const InfoBar = ({ difficulty, topic }: InfoBarProps) => {
+  return (
+    <HStack justifyContent={"space-between"}>
+      <HStack>
+        <UserInfo user={undefined} />
```

Comment:
you can use the `useAuth` hook to get the user for the first `UserInfo`

Reply:
Resolved in [36f0b62](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/pull/18/commits/36f0b62845752401a6dc0465480b5c62d929932c)